src/switch_rtp.c

```diff
@@ -2192,6 +2268,7 @@ static int check_rtcp_and_ice(switch_rtp_t *rtp_session)
 				rtcp_generate_report_block(rtp_session, rtcp_report_block, nack_dup);
 				rtp_session->rtcp_send_msg.header.count = 1; /* reception report block count */
 				stats->sent_pkt_count = 0;
+				sendRTCPEvent(rtp_session, sr, rtcp_report_block);
```

Comment:
call of sendRTCPEvent() should be under cond 'if (switch_channel_var_true(session->channel, "fire_rtcp_events")'

Reply:
@dragos-oancea please review our update.

Reply:
I didn't mean to tell you put the condition here, but where the RTP session is created.